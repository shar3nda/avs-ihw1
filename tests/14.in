10
4 -1 4 -2 -2 3 3 9 1 3