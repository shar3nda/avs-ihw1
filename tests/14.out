-2 2 3 -2 -9 4 6 4